### PR TITLE
GitHub Issue NOAA-EMC/GSI#152. 

### PR DESCRIPTION
**Pull request by lbi2018**
Update of modulefile.ProdGSI.wcoss_c and get_gefs_for_regional.f90 bug fixes.

### CODE REVIEW

**Review by TingLei-daprediction — APPROVED** [review]
Good to me! Thanks.

File: src/gsi/get_gefs_for_regional.f90 [hunk 1]
```diff
@@ -376,16 +376,14 @@ subroutine get_gefs_for_regional
...
-        ! FV3GFS write component does not include JCAP, infer from DIMY-2
-        njcap=latb-2
 
         nlat_gfs=gfshead%latb+2
         nlon_gfs=gfshead%lonb
         nsig_gfs=gfshead%levs
 
         jcap_gfs=gfshead%latb-2
 
-        if (mype==mype_out) write(6,*)'GESINFO:  Read NCEP FV3GFS netCDF ', &
```
Comment (MichaelLueken-NOAA) [inline src/gsi/get_gefs_for_regional.f90:388]: With the removal of mype_out on line 388/386, please remove the variable declaration on line 119.  This variable was noted as being unused while compiling in debug mode.

**Review by MichaelLueken-NOAA — COMMENTED** [review]
Hi Li,

I have completed my review of the changes in this PR.  The regression tests were run (both standard and debug) and all successfully passed.  It was noted, that while compiling in debug mode, one variable was flagged as unused.  When you replaced mype_out with 0, you will need to remove the declaration of mype_out on line 119 of get_gefs_for_regional.f90.  Once this is done, I will be able to get this work out to the review committee, with a due date of Monday.  If you have any questions, please let me know.

Mike

**Comment by lbi2018** [comment]
> Hi Li,
> 
> I have completed my review of the changes in this PR. The regression tests were run (both standard and debug) and all successfully passed. It was noted, that while compiling in debug mode, one variable was flagged as unused. When you replaced mype_out with 0, you will need to remove the declaration of mype_out on line 119 of get_gefs_for_regional.f90. Once this is done, I will be able to get this work out to the review committee, with a due date of Monday. If you have any questions, please let me know.
> 
> Mike

Hi Mike,

I removed line 119. 

Thanks,
-Li

File: src/gsi/get_gefs_for_regional.f90 (hunk 1 above)
Comment (MichaelLueken-NOAA) [reply src/gsi/get_gefs_for_regional.f90:388]: Thanks, Li!

**Comment by MichaelLueken-NOAA** [comment]
The due date has past with no feedback, so final approval will now be given and these changes will be merged to the NOAA-EMC/GSI master.

**Pull request merged**